decidim-templates/app/controllers/decidim/templates/admin/block_user_templates_controller.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Templates
+    module Admin
+      class BlockUserTemplatesController < Decidim::Templates::Admin::ApplicationController
+        include Decidim::TranslatableAttributes
+        include Decidim::Paginable
+
+        def fetch
+          enforce_permission_to :read, :template, template: template
+
+          response_object = {
+            template: translated_attribute(template.description)
+          }
+
+          respond_to do |format|
+            format.json do
+              render json: response_object.to_json
+            end
+          end
+        end
+
+        def copy
+          enforce_permission_to :copy, :template
+
+          CopyTemplate.call(template, current_user) do
+            on(:ok) do
+              flash[:notice] = I18n.t("templates.copy.success", scope: "decidim.admin")
+              redirect_to action: :index
+            end
+
+            on(:invalid) do
+              flash[:alert] = I18n.t("templates.copy.error", scope: "decidim.admin")
+              redirect_to action: :index
+            end
+          end
+        end
+
+        def destroy
+          enforce_permission_to :destroy, :template, template: template
+
+          DestroyTemplate.call(template, current_user) do
+            on(:ok) do
+              flash[:notice] = I18n.t("templates.destroy.success", scope: "decidim.admin")
+              redirect_to action: :index
+            end
+          end
+        end
+
+        def update
+          enforce_permission_to :update, :template, template: template
+          @form = form(TemplateForm).from_params(params)
+          UpdateTemplate.call(template, @form, current_user) do
+            on(:ok) do |template|
+              flash[:notice] = I18n.t("templates.update.success", scope: "decidim.admin")
+              redirect_to edit_block_user_template_path(template)
```

Comment:
I feel it's weird this redirects back to the edit path, although I understand it is in line with the other templates (i.e. survey templates).

The reason for survey templates to redirect back to the editing view is to get access to edit the survey itself. But it doesn't directly apply to this case as there is only the title and description to edit.

I think this is a UX quirk and this particular case should redirect back to the index.